pkg/apis/core/types.go

```diff
@@ -3691,6 +3691,15 @@ type PodStatus struct {
 	// +featureGate=DynamicResourceAllocation
 	// +optional
 	ResourceClaimStatuses []PodResourceClaimStatus
+
+	// TotalRequestedResources is the effective resource requests for this pod taking into consideration init
+	// and regular containers, and the pod overhead. When the InPlacePodVerticalScaling feature gate is enabled,
```

Comment:
To future-proof this comment when InPlacePodVerticalScaling is GA and feature gate is gone, can you please restate this?
Suggestion:
`If a resize of pod resources has been requested for a running pod, this field reflects the total desired resources for that pod which differs from actual resources allocated for the pod by the node.`

Reply:
@tzneal Besides the above suggestion, it looks good to me.
/lgtm in advance

Reply:
I agree, that's better wording, incorporated.